management/virtualmachine/client.go

```diff
@@ -648,8 +648,8 @@ func (self VirtualMachineClient) createServiceCertDeploymentConf(certPath string
 	return certConfig, nil
 }
 
-func (self VirtualMachineClient) createSshConfig(certPath, userName string) (SSH, error) {
-	sshConfig := SSH{}
+func (self VirtualMachineClient) createSshConfig(certPath, userName string) (*SSH, error) {
```

Comment:
no big deal. having all these pointers is just annoying. I can understand if we re using them for omitting things in XML marshaling but otherwise we shouldnt be doing any pointers, ideally. right?